### PR TITLE
feat: layout component updates

### PR DESCRIPTION
**Pull request by cindy-sf**

### CODE REVIEW

File: src/styles/theme.ts [hunk 1]
```diff
@@ -104,7 +105,7 @@ export const lightTheme: ThemeAttributes = {
   mode: themes.LIGHT,
   BACKGROUND: colors.WHITE,
   AVATAR_BORDER: colors.PURPLE,
-  INPUT_BACKGROUND: colors.GREY_LIGHT,
+  INPUT_BACKGROUND: colors.WHITE,
```
Comment (j0nathansf) [inline src/styles/theme.ts:108]: Why did you change this ?

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -1,45 +1,105 @@
...
+                width={50}
+                height={50}
+                src={headerOptions.userPicture.src}
+              />
+            </UserPictureWrapper>
+          )}
         </View>
       </Header>
-      {children}
+      <View style={{ alignItems: 'center', flex: 5 }}>{children}</View>
```
Comment (j0nathansf) [inline src/components/Layout/index.tsx]: Styled components 😀

**Review by j0nathansf — DISMISSED** [review]
I left you a few remarks.
Take them into account or not 👍

File: src/styles/theme.ts (hunk 1 above)
Comment (cindy-sf) [reply src/styles/theme.ts:108]: Because isn't the good color for this theme

**Review by j0nathansf — APPROVED** [review]
💹
